__tests__/pet.test.js

```diff
@@ -0,0 +1,203 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
+      const pet = new Pet('Fido');
+      expect(pet.name).toEqual('Fido');
+    });
+    it('has an initial age of 0', () => {
+      const pet = new Pet('Fido');
+      expect(pet.age).toEqual(0);
+    });
+    it('has initial hunger of 0', () =>{
+      const pet = new Pet('Fido');
+      expect(pet.hunger).toEqual(0);
+    });
+    it('has initial fitness of 10', () => {
+      const pet = new Pet('Fido');
+      expect(pet.fitness).toEqual(10);
+    });
+    it('has no children initially', () => {
+      const pet = new Pet('Fido');
+      expect(pet.children).toEqual([]);
+    })
+    it('is initially alive', () =>{
+      const pet = new Pet('Fido');
+      expect(pet.isAlive).toBe(true);
+    });
+  });
+  
+  describe('growUp', () => {
+    it('increments Fidos age by 1', () =>{
+      const pet = new Pet('Fido');
+      pet.growUp();
+      expect(pet.age).toEqual(1);
+    });
+    it('increments Fidos age by 2 if we call the function 2 times', () =>{
+      const pet = new Pet('Fido)');
+      pet.growUp();
+      pet.growUp();
+      expect(pet.age).toEqual(2);
+    });
+  });
+
+  describe('hunger', ()=> {
+    it('increments Fidos hunger by 5 on aging by 1', () =>{
+      const fido = new Pet('Fido');
+      fido.growUp();
+      expect(fido.hunger).toEqual(5);
+    });
+  });
+
+  describe('fitness', () => {
+    it('decreases Fidos fitness by 3 on aging by 1', () =>{
+      const fido = new Pet('Fido');
+      fido.growUp();
+      expect(fido.fitness).toEqual(7);
+    });
+  });
+
+  describe('walk', () => {
```

Comment:
This line should not be indented at all, remember to keep the level of indent consistent across the repo (4 spaces in this case). This should be fixed for the remainder of this file